### ISSUE TITLE
CDO in Julia

### ISSUE THREAD

**Issue by Datseris** [JuliaClimate/meta#8]
Has anyone here written some kind of interface that allows using CDO https://code.mpimet.mpg.de/projects/cdo/wiki in Julia?

Should we do that? Seems to be in the same alley as ClimateTools.jl @Balinus  . 

(I want to use it for grid remapping, but maybe someone also tells me if there is already grid remapping implemented)

**Comment by gaelforget** [comment]
First time I hear about this one (thanks @Datseris !). 

Maybe a simple first step would be to just use pycall towards their python frontend. I did this with a couple other things ([argovis](https://argovis.colorado.edu/) in https://github.com/gaelforget/ArgoData.jl and [cmap](https://cmap.readthedocs.io/en/latest/) in https://github.com/gaelforget/MarineEcosystemNotebooks) in case that helps.

**Comment by Datseris** [comment]
Yeah my first step was also to use the python interface, but unfortunately they don't support Windows not even through the Python interface....... :( :( :( :( :( The manners of those people are unbelievable!
